cpp/include/raft/spatial/knn/specializations.cuh

```diff
@@ -23,6 +23,4 @@
 #include <raft/neighbors/specializations/fused_l2_knn.cuh>
 #include <raft/neighbors/specializations/knn.cuh>
 
-#include <raft/neighbors/specializations/detail/ivf_pq_search.cuh>
```

Comment:
Ah, yeah we definitely don't want to include the specializations header for the function we are compiling- that will make it not compile it at all. Good catch!